crates/ruff_linter/resources/test/fixtures/wemake_python_styleguide/control_var_used_after_block.py

```diff
@@ -0,0 +1,90 @@
+for global_var in []:
```

Comment:
Friendly ping @charliermarsh :bow: 

Just looking for the next steps here (whether that be in a queue, pinging another person, etc)